### ISSUE TITLE
Execute The Python file twice in a row produce duplicate entry. (Bug)

### ISSUE THREAD

**Issue by WindDrifter** [vancouvercommunitynetwork/donation-import#6]
I execute the main.py three times (python main.py command in linux)
When I check the tables in My sql: This occurs:

+-----------+-------+--------------+------------+----------+------+---------------+---------------+-------------+-------------+-------------+---------------+
| Receipt # | Id #  | Amount Payed | Date Payed | For      | Cash | Receipt Given | Main Login Id | Login Id #2 | Login Id #3 | Login Id #4 | Paper Receipt |
+-----------+-------+--------------+------------+----------+------+---------------+---------------+-------------+-------------+-------------+---------------+
|     32392 | 15230 |           10 | NULL       | Donation |    0 |          NULL | 000           | NULL        | NULL        | NULL        | NULL          |
|     32393 | 15230 |           10 | NULL       | Donation |    0 |          NULL | 000           | NULL        | NULL        | NULL        | NULL          |
|     32394 | 15230 |           10 | NULL       | Donation |    0 |          NULL | 000           | NULL        | NULL        | NULL        | NULL          |
+-----------+-------+--------------+------------+----------+------+---------------+---------------+-------------+-------------+-------------+---------------+

As you can see: There are 3 entries by using the same csv file. 

TL:DR: No stopper for same csv files, causing duplicate entries.

Edit1: Issue occur due to parsing issue as you can see Date Payed is shows as 0 for all 3 of them.
Edit2: I am using Python 2.7.
- Warning shows when executing the program: Warning: Incorrect datetime value: '2015-06-04' for function str_to_date
  dbCursor.execute(sql, (donorID, donorInfo.amountPaid, donorInfo.datePaid))

**Comment by RyanCuaVCN** [comment]
Issue fixed via duplicated issue fix pull request.
